python/cuml/manifold/umap.pyx

```diff
@@ -299,7 +298,8 @@ class UMAP(Base):
         cdef UMAPParams* umap_params = <UMAPParams*>params_t
 
         state['X_m'] = cudf.DataFrame.from_gpu_matrix(self.X_m)
-        state['arr_embed'] = cudf.DataFrame.from_gpu_matrix(self.arr_embed)
+        state['embedding_'] = cudf.DataFrame.from_gpu_matrix(self.embedding_)
```

Comment:
Do we want to keep this a df in the long run or move to a gpu array in the future?

Reply:
This is just for picking, since a dataframe is serializable and numba arrays are not. This is being done in all of our algorithms.